index.html

```diff
@@ -5007,13 +4983,259 @@ <h3 id="looping-AudioBufferSourceNode">
+            frame is interpolated. Of particular note is the last sample frame
+            in the looped output, which is interpolated using the loop start
+            point:
+          </p>
+          <figure>
+            <img alt="AudioBufferSourceNode playbackRate interpolation" src=
+            "images/absn-slow-loop.png" width="543" height="472">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> playbackRate interpolation
+            </figcaption>
+          </figure>
+          <p>
+            Sample rate interpolation, illustrating playback of a buffer whose
+            sample rate is 50% of the context sample rate, resulting in a
+            computed playback rate of 0.5 that corrects for the difference in
+            sample rate between the buffer and the context:
+          </p>
+          <figure>
+            <img alt="AudioBufferSourceNode sample rate interpolation" src=
+            "images/absn-half-sample-rate.png" width="543" height="459">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> sample rate interpolation.
+            </figcaption>
+          </figure>
           <p>
-            When scheduling the beginning and the end of playback using the
-            <code>start()</code> and <code>stop()</code> methods, the resulting
-            start or stop time MUST be rounded to the nearest sample-frame in
-            the sample rate of the <a><code>AudioContext</code></a>. That is,
-            no sub-sample scheduling is possible.
+            Subsample offset playback, in which the offset within the buffer
+            begins at exactly half a sample frame. Consequently, every output
+            frame is interpolated:
           </p>
+          <figure>
+            <img alt="AudioBufferSourceNode subsample offset playback" src=
+            "images/absn-offset.png" width="543" height="448">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> subsample offset playback
+            </figcaption>
+          </figure>
+          <p>
+            Reversed loop playback, showing how negative playback rates
+            interact with looped output, showing that reversed playback is an
+            exact reversal of what would occur with forward playback including
+            the pre-loop portion of the buffer followed by a specific number of
+            loop iterations:
+          </p>
+          <figure>
+            <img alt="AudioBufferSourceNode reversed loop playback" src=
+            "images/absn-reversed-loop.png" width="550" height="451">
+            <figcaption>
+              <a>AudioBufferSourceNode</a> reversed loop playback
+            </figcaption>
+          </figure>
+          <p>
+            Subsample loop playback, showing how fractional frame offsets in
+            the loop endpoints map to interpolated data points in the buffer
+            that respect these offsets as if they were references to exact
+            sample frames:
```

Comment:
Actually the looped sections are suppsed to be identical copies although an editing error displaced one of the repeats vertically by mistake. I don't know why the repeats would be different in any way since the loop length is an exact multiple of the sample interval.

I did discover that the samples were marked wrongly in this diagram as interpolated vs. exact, and fixed that. I also updated all the diagrams to show the same distinction in the output signal (not just in the buffer contents) which I feel makes it all clearer.